### DISCUSSION TITLE
FDRS LoRa settings

### DISCUSSION THREAD

**Discussion by javrolijk** [timmbogner/Farm-Data-Relay-System#124] [Q&A]
I was wondering what the reason would be for having a LoRa setting section in the 'fdrs_globals.h' file and all Example files? Below is a screenshot as a reference.

![image](https://user-images.githubusercontent.com/70583691/205188626-1ea46bc9-7c94-468b-8cbd-5df5f01a2ebb.png)

**Comment by timmbogner** [comment]
The idea is that if the user wants to, they can define any setting in the config file instead of using the globals. In truth it goes back to before there was a globals file, and every config was always defined in the config. Even after globals was added, you didn't need to install anything else, so the user had the option to install the globals, but might not do it.

I've been thinking of how to re-think configuration. Now that the gateway INO is devoid of most code, it may be an option to move the essential configs to the INO and get rid of the config file. We'll see about that.

Looking at it now, TXPWR should not be a global, since it doesn't need to be the same across the system. There are a few more more parameters I think I'll add/change while I'm working with RadioLib.

Check out my pull request if you haven't seen it yet!
